targets/targets.json

```diff
@@ -9047,8 +9057,16 @@
         "sectors": [[268443648, 512]],
         "bootloader_supported": true
     },
-    "CY8CPROTO_064_SB": {
-        "inherits": ["MCU_PSOC6_M4"],
+    "CY8CPROTO_064_SB_S": {
+        "inherits": ["PSA_DUAL_V7_M_SPE"],
+        "tfm_target_name": "psoc64",
+        "tfm_bootloader_supported": false,
+        "tfm_default_toolchain": "ARMCLANG",
+        "tfm_supported_toolchains": ["ARMCLANG", "GNUARM"],
+        "delivery_dir": "TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CPROTO_064_SB/prebuilt"
```

Comment:
Why not directly into `TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CPROTO_064_SB/`?

Reply:
@Patater continuing the tradition of putting binaries in a dedicated folder.

Reply:
It's a tradition now, is it?

Mbed Bootloader doesn't do this.
`./features/FEATURE_BOOTLOADER/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/TARGET_FRDM/mbed-bootloader-k66f-block_device-sotp-v3_4_0.bin`

Softdevice doesn't do this.
`./targets/TARGET_NORDIC/TARGET_MCU_NRF51822/device/TOOLCHAIN_IAR/s110_nrf51822_7.1.0_softdevice.bin`

Cypress WHD doesn't do this.
`./targets/TARGET_Cypress/TARGET_PSOC6/COMPONENT_WHD/resources/firmware/COMPONENT_4343W/4343WA1-mfgtest.bin`

It looks like maybe Mbed PSA used to do this, but I think it's better style not to have a `prebuilt` folder.

Reply:
Fixed.